docs/geedocs/answer/2987876.html

```diff
@@ -45,6 +61,7 @@
 <li>MrSID</li>
 <li>numpy</li>
 <li>OGDI</li>
```

Comment:
This list has corresponding 3rd party licenses in https://github.com/google/earthenterprise/tree/master/earth_enterprise/third_party ?

I don't see few of them. Ex: OGDI, gif, etc.

Reply:
See the above answer as to why